javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -1434,7 +1433,7 @@ i18n.phonenumbers.PhoneNumberUtil.prototype.isNumberGeographical =
 /**
  * Helper function to check region code is not unknown or null.
  *
- * @param {?string} regionCode the ISO 3166-1 two-letter region code.
+ * @param {?string} regionCode the CLDR two-letter region code.
```

Comment:
Nope, minus REGION_CODE_FOR_NON_GEO_ENTITY. 
I guess in the NANPA one it could be, we will just say "no" as we would if you passed in "lara" or "null" :)